features/bootstrap/DoctrineContext.php

```diff
@@ -1227,6 +1232,108 @@ public function thereAreNbDummyDtoCustom($nb)
         $this->manager->clear();
     }
 
+    /**
+     * @Given there is an order with same customer and recipient
+     */
+    public function thereIsAnOrderWithSameCustomerAndRecipient()
+    {
+        $customer = $this->isOrm() ? new Customer() : new CustomerDocument();
+        $customer->name = 'customer_name';
+
+        $address1 = $this->isOrm() ? new Address() : new AddressDocument();
+        $address1->name = 'foo';
+        $address2 = $this->isOrm() ? new Address() : new AddressDocument();
+        $address2->name = 'bar';
+
+        $order = $this->isOrm() ? new Order() : new OrderDocument();
+        $order->recipient = $customer;
+        $order->customer = $customer;
+
+        $customer->addresses->add($address1);
+        $customer->addresses->add($address2);
+
+        $this->manager->persist($address1);
+        $this->manager->persist($address2);
+        $this->manager->persist($customer);
+        $this->manager->persist($order);
+
+        $this->manager->flush();
+        $this->manager->clear();
+    }
+
+    /**
+     * @Given there are :nb sites with internal owner
+     */
+    public function thereAreSitesWithInternalOwner(int $nb)
+    {
+        for ($i = 1; $i <= $nb; ++$i) {
+            $internalUser = new \ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\InternalUser();
```

Comment:
Why not use?

Reply:
Didn't notice this as this commit was taken from the other PR. But I wonder why PHP-CS-Fixer doesn't catch this... :thinking: